Continue Create info module for each DSO (#15)
mdavidsaver/setuptools_dso: README.md

)
```

### Cython

Version 1.3 added a `setuptools_dso.cythonize()` wrapper to correctly handle `Extension(dso=...)`.

### Runtime

Beginning with setuptools-dso 2.0 a file `*_dso.py` will be generated alongside each DSO.

> eg. dso `mypkg.lib.thelib` will create `mypkg/lib/thelib_dso.py`.

Reviewer: ```suggestion
eg. dso `mypkg.lib.thelib` will create `mypkg/lib/thelib_dsoinfo.py`.
```

If `mypkg.lib` is a valid python packages (contains `__init__.py`)

> then `mypkg.lib.thelib_dso.py` can be imported, and will contain the attributes:

Reviewer: ```suggestion
then `mypkg.lib.thelib_dsoinfo` can be imported, and will contain the attributes:
```

- `.dsoname`    eg. "mypkg.lib.thelib"
- `.libname`    eg. "thelib.so"
- `.soname`     eg. "thelib.so.0"
- `.filename`   eg. "/full/path/to/thelib.so"
- `.sofilename` eg. "/full/path/to/thelib.so.0"

> Beginning with 2.0 `os.add_dll_directory()` is called by the generated `*_dso.py` module.

Reviewer: ```suggestion
Beginning with 2.0 `os.add_dll_directory()` is called by the generated `*_dsoinfo.py` module.
```

### Runtime

Beginning with setuptools-dso 2.0 a file `*_dso.py` will be generated alongside each DSO.
eg. dso `mypkg.lib.thelib` will create `mypkg/lib/thelib_dso.py`.
If `mypkg.lib` is a valid python packages (contains `__init__.py`)

> then `mypkg.lib.thelib.dso.py` can be imported, and will contain the attributes:

Reviewer: ```suggestion
then `mypkg.lib.thelib_dso.py` can be imported, and will contain the attributes:
```

- `.dsoname`    eg. "mypkg.lib.thelib"
- `.libname`    eg. "thelib.so"
- `.soname`     eg. "thelib.so.0"
- `.filename`   eg. "/full/path/to/thelib.so"
- `.sofilename` eg. "/full/path/to/thelib.so.0"

Beginning with 2.0 `os.add_dll_directory()` is call by the generated `*_dso.py` module.

> Prior to 2.0, or if the generate moduls is not used,

Reviewer: ```suggestion
Prior to 2.0, or if the generated module is not used,
```

Author: Yes yes, my speling is treible :)
